Vagrantfile

```diff
@@ -20,6 +20,10 @@ openattic_docker_repo = settings.has_key?('openattic_docker_repo') ?
                         settings['openattic_docker_repo'] :
                         'https://github.com/openattic/openattic-docker.git'
 
+openattic_docker_is_local = settings.has_key?('openattic_docker_is_local') ?
```

Comment:
What about checking if the existing "openattic_docker_repo" property starts with "http://", "https://" or "git@" (if yes, then it's a remote repository, else it's a local repository)?

This will prevent us from adding a new property ("openattic_docker_is_local") to our settings.